frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -23,52 +23,56 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
   disableDeleteRow,
   disableAddRow,
   toolTip,
-}) => (
-  <FieldArray
-    name={name}
-    render={({ push, remove, form }) => {
-      const fieldValue = _.get(form.values, name, []);
-      return (
-        <FormGroup
-          fieldId={`form-multi-column-input-${name.replace(/\./g, '-')}-field`}
-          label={label}
-          helperText={helpText}
-          isRequired={required}
-        >
-          {fieldValue.length < 1 ? (
-            emptyMessage && (
-              <div className="odc-multi-column-field__empty-message">
-                <SecondaryStatus status={emptyMessage} />
-              </div>
-            )
-          ) : (
-            <MultiColumnFieldHeader headers={headers} />
-          )}
-          {fieldValue.length > 0 &&
-            fieldValue.map((value, index) => (
-              <MultiColumnFieldRow
-                key={`${index.toString()}`} // There is no other usable value for key prop in this case.
-                name={name}
-                toolTip={toolTip}
-                rowIndex={index}
-                onDelete={() => remove(index)}
-                isReadOnly={isReadOnly}
-                disableDeleteRow={disableDeleteRow}
-              >
-                {children}
-              </MultiColumnFieldRow>
-            ))}
-          {!isReadOnly && (
-            <MultiColumnFieldFooter
-              disableAddRow={disableAddRow}
-              addLabel={addLabel}
-              onAdd={() => push(emptyValues)}
-            />
-          )}
-        </FormGroup>
-      );
-    }}
-  />
-);
+}) => {
+  const { values } = useFormikContext<FormikValues>();
+  const fieldValue = _.get(values, name, []);
+  useFormikValidationFix(fieldValue);
```

Comment:
Doesn't this put a requirement on `name` being the one that is changed? What if you have a MultiColumnField with multiple required fields?

Test this by re-enabling the requirement on default. I suspect if you have a name, but edit default to be empty you can still submit.

Reply:
@andrewballantyne multicoulmField internally uses FieldArray from formik which accepts an key and value as array(array of objects).
eg: 
` { 
        parameters: [{name: 'test', description: '', default: 'value'}]
}`
here the `name` key will be `parameters/resources`, the values will be the array of objects which can support multiple required fields.

Reply:
That's the name of the whole formik field itself that manages the specific part of the form state. It was there before as well. Nothing to do with specific input fields.

Reply:
Ah I see - it's actually the values... I saw "name" and I assumed it was the field name, not the fieldArray name... We are good 😄 my bad.